app/views/admin/action_pages/index.html.erb

```diff
@@ -5,7 +5,28 @@
   <%= form_tag admin_action_pages_path, id: "filter_action_pages", method: :get do %>
     <div class="form-item">
       <%= label_tag "Type" %>
-      <%= select_tag :action_type, options_for_select(%w(all call congress_message email petition tweet redirect)) %>
+      <%= select :action_filters, :type, options_for_select(%w(all call congress_message email petition tweet redirect)) %>
+    </div>
+
+    <div class="form-item">
+      <%= label_tag "Date Range" %>
+      <input type="text" class="form-control text-inline" id="date_range" name="date_range"
```

Comment:
The `form-control text-inline` classes are from the old stylesheet and aren't needed anymore.